### ISSUE TITLE
About some statements in KNOWLEDGE FUSION OF LARGE LANGUAGE MODELS 

### ISSUE THREAD

**Issue by bw-wang19** [18907305772/FuseAI#24]
Dear authors, I have some problems while reading the paper KNOWLEDGE FUSION OF LARGE LANGUAGE MODELS, I noticed there are two formats discribed the losses designed for training.  I suppose the loss function should be minimized while the discrepancy function D(·) is minimized, but these formats show an opposite result?
  
![image](https://github.com/user-attachments/assets/a0788346-f676-4a1e-b025-73e57b711afe)
![image](https://github.com/user-attachments/assets/e2de1582-f886-4cd1-8587-bee4167b6263)
